mlflow/tracking/client.py

```diff
@@ -1234,6 +1234,7 @@ def log_figure(
         run_id: str,
         figure: Union["matplotlib.figure.Figure", "plotly.graph_objects.Figure"],
         artifact_file: str,
+        **kwargs,
```

Comment:
```suggestion
        *,
        kwargs,
```

Can we make this a separate argument? In the future, we might add a new argument to this function. If that argument conflicts with plotly or matplotlib, that's an issue.

Reply:
I see, good point! Should we then also rename `kwargs` to sth. more expressive, e.g. `figure_args` or `plotting_args` (open to other suggestions), so that it is clear that those arguments are being passed to the plotting backend?

Reply:
`xxx_args` sounds like it takes a tuple. I'd prefer `xxx_kwargs`.

- `figure_kwargs`
- `plot_kwargs`
- `extra_kwargs`
- `save_kwargs`

Reply:
`kwargs` doesn't sound bad because we probably won't add an argument named `kwargs` to `log_figure`.

Reply:
`save_kwargs` sound good, the other suggestions like `figure_kwargs` might cause confusion as it could be interpreted as altering the figure itself. Also sticking with `kwargs` if fine by me :) 
Is there maybe another (similar) case in the mlflow api where we could reuse the naming convention for additional kwargs passed to the backend?

Reply:
I also like `save_kwargs`. We have `saved_model_kwargs` here:

https://github.com/mlflow/mlflow/blob/0e856e297023b3e2ae64438344463b393d70ac8d/mlflow/tensorflow/__init__.py#L199